Cargo.toml

```diff
@@ -15,3 +15,8 @@ codecov = { repository = "rust-cli/clap-verbosity-flag" }
 [dependencies]
 log = "0.4.1"
 structopt = "0.3"
+paw = { version = "1.0", optional = true }
+
+[features]
+default = []
+paw-compatibility = ["paw"]
```

Comment:
> I had an error to use this library in combination with structopt + paw, so I added paw as an optional dependency behind a feature.

How does having a `paw` dependency fix our support for `paw`?

Reply:
I added one demonstration as a new commentary, and I'm also now repeating what I placed in there, but I think that this is the case because on how structopt macros references paw.